src/main/java/org/terasology/launcher/game/TerasologyGameVersions.java

```diff
@@ -64,12 +64,18 @@
 
     private final Map<GameJob, List<TerasologyGameVersion>> gameVersionLists;
     private final Map<GameJob, SortedMap<Integer, TerasologyGameVersion>> gameVersionMaps;
-    private final Gson gson;
+
+    private final IBuildRepository buildRepository;
 
     public TerasologyGameVersions() {
+        this(new OnlineBuildRepository());
+    }
+
+    public TerasologyGameVersions(IBuildRepository buildRepository) {
         gameVersionLists = new EnumMap<>(GameJob.class);
         gameVersionMaps = new EnumMap<>(GameJob.class);
-        gson = new Gson();
+        assert buildRepository != null;
```

Comment:
I'm not sure that `assert` is the right thing to use here. There's also `Preconditions....` to check that some object is non-null (that would be prefered here).